tests/test_screen.py

```diff
@@ -936,6 +936,45 @@ def signal_check(self, screen):
         """Dummy callback for screen wrapper."""
         self.assertEqual(signal.getsignal(signal.SIGWINCH), screen._resize_handler)
 
+    def test_overflow_decorator(self):
+        """
+        Check underflow and overflow work as expected
+        """
+        @Screen.Session
+        def demo(screen=None):
+            canvas = Canvas(screen, 10, 40, 0, 0)
+
+            # Check underflow and overflow work as expected
+            canvas.print_at("ab", -1, 0)
+            canvas.print_at("cd", canvas.width - 1, 0)
+            self.assert_line_equals(canvas, "b                                      c")
+
+        demo()
+
+    @Screen.Session
+    def test_overflow_class_decorator(self, screen=None):
+        """
+        Check underflow and overflow work as expected
+        """
+        canvas = Canvas(screen, 10, 40, 0, 0)
+
+        # Check underflow and overflow work as expected
+        canvas.print_at("ab", -1, 0)
+        canvas.print_at("cd", canvas.width - 1, 0)
+        self.assert_line_equals(canvas, "b                                      c")
+
+    def test_overflow_with(self):
+        """
+        Check underflow and overflow work as expected
```

Comment:
Could you please update the comment to explain that it is testing that the ScreenSession class works as a context manager?